piksi_tools/bootload.py

```diff
@@ -88,6 +105,10 @@ def wait_for_handshake(self, timeout=None):
     return True
 
   def reply_handshake(self):
+    """
+    Send a handshake to Piksi bootloader to request bootloader mode. Should be
+    sent after receiving handshake from Piksi bootloader.
+    """
     # < 0.45 of SBP protocol, reuse single handshake message.
     if self.sbp_version < (0, 45):
       self.link.send(SBP_MSG_BOOTLOADER_HANDSHAKE_DEPRECATED, '\x00')
```

Comment:
@mookerji Is there an example for how to use the message class to send messages, rather than decoding a received message in a callback?